assets/htlc/script.go

```diff
@@ -0,0 +1,88 @@
+package htlc
```

Comment:
could we add a tests for these similar to https://github.com/lightninglabs/loop/blob/master/staticaddr/script/script_test.go?